lib/ansible/plugins/connection/winrm.py

```diff
@@ -76,7 +77,7 @@ def set_host_overrides(self, host):
         '''
         Override WinRM-specific options from host variables.
         '''
-        host_vars = host.get_vars()
+        host_vars = combine_vars(host.get_group_vars(), host.get_vars())
```

Comment:
That was my first thought upon looking at this as well, but if I remember right the problem is we don't pass the variables down below the level of the action plugin anymore, so this was the workaround for that.